paddle/operators/CMakeLists.txt

```diff
@@ -43,33 +45,32 @@ endfunction()
 
 add_subdirectory(math)
 
-cc_test(gather_test SRCS gather_test.cc DEPS tensor)
-op_library(gather_op SRCS gather_op.cc gather_op.cu)
-
-cc_test(scatter_test SRCS scatter_test.cc DEPS tensor)
-op_library(scatter_op SRCS scatter_op.cc scatter_op.cu)
-
-cc_library(net_op SRCS net_op.cc DEPS op_registry)
-cc_test(net_op_test SRCS net_op_test.cc DEPS net_op)
-
-op_library(add_op SRCS add_op.cc add_op.cu)
+set(OP_WITHOUT_DEPS
```

Comment:
Sorting op names manually seems inconvenient and can be wrong of order. Maybe we can:

1. find op source files under specific folder: `ALL_OPS`
1. define a list containing ops that need to setup manually, including `FRAMEWORK_OP`(net_op), and ops need `OP_DEPS`
1. exclude `FRAMEWORK_OP` and `OP_DEPS` from `ALL_OPS`, generating `GENERAL_OPS`
1. sort `GENERAL_OPS`
1. call `op_library`

Reply:
Thx for your comment, I will refine it.
But `net_op` is similar to other ops, so we don't need `FRAMEWORK_OP`